### PR TITLE
Feature/ios plugin

### PR DESCRIPTION
**Pull request by Labun**

### CODE REVIEW

File: ios/Classes/SwiftSchibstedAccountPlugin.swift [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        guard let clientConfiguration = clientConfiguration else { return }
+        
+        let config = IdentityUIConfiguration(clientConfiguration: clientConfiguration)
+        self.identityUI = IdentityUI(configuration: config)
+        self.identityUI?.delegate = self
+        guard let viewCtrl = UIApplication.shared.delegate?.window??.rootViewController else {
+            return
+        }
+        
+        self.identityUI?.presentIdentityProcess(from: viewCtrl, route: IdentityUI.Route.login)
```
Comment (mbmadaj) [inline ios/Classes/SwiftSchibstedAccountPlugin.swift]: as it's enum - route could be simply `.login`

File: ios/Classes/SwiftSchibstedAccountPlugin.swift [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            onUserLoggedIn(user: user)
+        default: break
+        }
+    }
+    
+    private func login() {
+        guard let clientConfiguration = clientConfiguration else { return }
+        
+        let config = IdentityUIConfiguration(clientConfiguration: clientConfiguration)
+        self.identityUI = IdentityUI(configuration: config)
```
Comment (mbmadaj) [inline ios/Classes/SwiftSchibstedAccountPlugin.swift]: I think that you like to omit `self.` unless it's necessary - here it's there to be removed 😉

File: ios/Classes/SwiftSchibstedAccountPlugin.swift [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    
+    private func logout() {
+        user?.logout()
+        sendEvent(SchibstedAccountEventFactory.loggedOut)
+    }
+    
+    private func onUserLoggedIn(user: SchibstedAccount.User) {
+        self.user = user
+        
+        sendEvent(SchibstedAccountEventFactory.loggedIn(user: nil, userProfile: nil))
```
Comment (mbmadaj) [inline ios/Classes/SwiftSchibstedAccountPlugin.swift:91]: is this event really necessary?

File: ios/Classes/Models/SchibstedAccountEvent.swift [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+struct SchibstedAccountEvent: Codable {
+    let schibstedAccountLoginState: String
```
Comment (mbmadaj) [inline ios/Classes/Models/SchibstedAccountEvent.swift:4]: Would be great to not have these prefixes (`schibstedAccount`) for each variable. However I don't have knowledge if these are required when sending json, if yes then please discard this comment.

File: ios/Classes/SwiftSchibstedAccountPlugin.swift (hunk 1 above)
Comment (Labun) [reply ios/Classes/SwiftSchibstedAccountPlugin.swift]: 👍

File: ios/Classes/SwiftSchibstedAccountPlugin.swift (hunk 2 above)
Comment (Labun) [reply ios/Classes/SwiftSchibstedAccountPlugin.swift]: 👍

File: ios/Classes/SwiftSchibstedAccountPlugin.swift (hunk 3 above)
Comment (Labun) [reply ios/Classes/SwiftSchibstedAccountPlugin.swift:91]: yep, business need

File: ios/Classes/Models/SchibstedAccountEvent.swift (hunk 4 above)
Comment (Labun) [reply ios/Classes/Models/SchibstedAccountEvent.swift:4]: the prefix added for sake of consistency with android version

**Comment by kawa89** [comment]
@Labun can this be merged?

**Comment by Labun** [comment]
@kawa89 Yep, I added fixes suggested by @mbmadaj and ios readme

**Review by kawa89 — APPROVED** [review]

**Pull request merged**